integration-tests/tests/src/utils/import-app.ts

```diff
@@ -56,8 +56,10 @@ export function getDefaultReplacements(name: string): TemplateReplacements {
   // When running on CI we connect through mongodb-atlas instead of local-mongodb
   const { mongodbClusterName } = environment;
   if (typeof mongodbClusterName === "string") {
+    const replacements = { "config.json": { name: `${name}-${mongodbClusterName}` } };
```

Comment:
Could we move this to a separate `appName` const and reuse that below?
I remember Nikola mentioning that the app name needs to be below 31 symbols - how do we guarantee that?

Reply:
I remember Nikola mentioning that the app name needs to be below 31 symbols - how do we guarantee that?

Reply:
We could throw an exception 🤷🏼 That's in our hands to create valid app names.  The clusterName.length should stay consistent in CI.